src/controlP5/ControlWindow.java

```diff
@@ -843,7 +843,7 @@ public ControlWindow setUndecorated( boolean theFlag ) {
 			_myApplet.frame.removeNotify( );
 			_myApplet.frame.setUndecorated( isUndecorated );
 			_myApplet.setSize( _myApplet.width , _myApplet.height );
-			_myApplet.setBounds( 0 , 0 , _myApplet.width , _myApplet.height );
+			//_myApplet.setBounds( 0 , 0 , _myApplet.width , _myApplet.height );
```

Comment:
Ran into this too. Might be better off deleting it. For anyone unfamiliar, PApplet.setBounds is not present anymore and the project won't compile without this line commented out or deleted.